src/extension/ncwms/src/main/java/au/org/emii/geoserver/wms/Ncwms.java

```diff
@@ -112,8 +112,9 @@ private void proxyWmsRequest(HttpServletRequest request, HttpServletResponse res
             if (isCollectionWithTimeMismatch(layerDescriptor.geoserverName())) {
                 wmsParameters.remove("TIME");
             }
-
+            wmsParameters.remove("version");
             wmsParameters.put("VERSION", new String[] { wmsVersion });
```

Comment:
URLS's ended up with version=1.1.0 and VERSION="1.3.0" for example and the order of parameters added are also chaotic thanks to Java. If we definitely want to add `VERSION` we should then remove `version`